inventory_management_system_api/models/item.py

```diff
@@ -8,6 +8,8 @@
 from inventory_management_system_api.models.catalogue_item import Property
 from inventory_management_system_api.models.custom_object_id_data_types import CustomObjectIdField, StringObjectIdField
 
+# pylint: disable=duplicate-code
```

Comment:
What is the other class that's detected as a duplicate of `ItemIn`?

Reply:
See https://github.com/ral-facilities/inventory-management-system-api/actions/runs/7654698135/job/20859204813, its the notes field all the way down to the end of `validate_properties` I could do it just for that, but I thought it would be weird in the middle of the properties.

Reply:
Oh right, I thought it was Pylint getting confused.